conga-aem-definitions/src/main/templates/aem-dispatcher-cloud/conf.d/variables/global.vars.hbs

```diff
@@ -34,5 +37,5 @@ Define REWRITE_LOG_LEVEL {{httpd.logging.rewriteLogLevel}}
 
 {{#httpdCloudManagerConditional}}
 # Set the default expiration time for text/html responses (in minutes)
```

Comment:
The comment should also mention "seconds"

Reply:
good catch, fixed